src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```diff
@@ -164,8 +166,8 @@
   word-wrap: break-word;
   background-color: var(--pf-c-clipboard-copy__expandable-content--BackgroundColor);
   background-clip: padding-box;
-  border: var(--pf-c-clipboard-copy__expandable-content--BorderWidth) solid transparent;
-  outline-offset: var(--pf-c-clipboard-copy__expandable-content--OutlineOffset);
+  border: solid var(--pf-c-clipboard-copy__expandable-content--BorderColor);
+  border-width: 0 var(--pf-c-clipboard-copy__expandable-content--BorderRightWidth) var(--pf-c-clipboard-copy__expandable-content--BorderBottomWidth) var(--pf-c-clipboard-copy__expandable-content--BorderLeftWidth);
```

Comment:
Also, makes sense to add `--pf-c-clipboard-copy__expandable-content--BorderTopWidth` so it's themeable.

Reply:
Updated.